### ISSUE TITLE
Delete multiple access levels

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#39656]
### Steps to reproduce the issue
Create three access levels (1,2,3)
Create an article at level 2
Create a contact at level 3

## Test 1
Select all three access and try to delete them

### Expected result
Level 1 is deleted and an error message that I cannot delete 2 and 3 because they are in use

### Actual result
Level 1 is deleted and an error message that I cannot delete 2  because it is in use. No mention of 3

**Comment by wojtekxtx** [comment]
@brianteeman title is highly misleading. Suggest rewording it. Besides: why would you want to remove AL's? Wouldnt be better to just rename them?

**Comment by brianteeman** [comment]
What would you suggest as a better title?

If you can add something then you should be able to delete it as well. But that decision has already been made. This issue is about how it works or doesn't work

**Comment by wojtekxtx** [comment]
> What would you suggest as a better title?

@brianteeman Maybe "delete not used/not enabled access levels"?

**Comment by chmst** [comment]
The focus is on "multiple". And the access leveles are used - this is why the message appears.

**Comment by brianteeman** [comment]
This is what I expected
![image](https://user-images.githubusercontent.com/1296369/213295031-1b0df890-d5f9-49d9-86d9-048821585a9b.png)

This is what I got

![image](https://user-images.githubusercontent.com/1296369/213295078-cc021d3d-f38c-4404-9e9a-082a56e6b6cd.png)

**Comment by wojtekxtx** [comment]
@brianteeman & @chmst everything that user is, in any way, able to interact with, should be deletable.

**Comment by brianteeman** [comment]
You really don't have a clue what we are talking about
